OtripleS.Web.Api/Models/Students/Student.cs

```diff
@@ -4,6 +4,22 @@ namespace OtripleS.Web.Api.Models.Students
 {
     public class Student : IAuditable
     {
+        public Student()
+        {
+        }
+
+        public Student(Guid id, string firstName, string middleName, string lastName, DateTimeOffset birthDate, Gender gender)
+        {
+            Id = id;
+            FirstName = firstName;
+            MiddleName = middleName;
+            LastName = lastName;
+            BirthDate = birthDate;
+            Gender = gender;
+
+            CreatedDate = DateTime.UtcNow;
```

Comment:
new empty lines are not needed here

Reply:
ok, the rationale was to separate client specified fields and system fields, but you are right, it is not needed, will remove